en/micro-integrator/docs/references/connectors/amazons3-connector/amazons3-connector-example.md

```diff
@@ -0,0 +1,308 @@
+# Amazon S3 Connector Example 
+
+The AmazonS3 Connector allows you to access the REST API of Amazon Simple Storage Service (Amazon S3), which provides a simple web services interface that can be used to store and retrieve any amount of data, at any time, from anywhere on the web. It gives any developer access to the same highly scalable, reliable, secure, fast, inexpensive infrastructure that Amazon S3 uses to run its own global network of websites. The service aims to maximize benefits of scale and to pass on those benefits to developers.
+
+Hence WSO2 EI AmazonS3 Connector is useful to take your on-premise data to the cloud. The advantage is, you do not need to worry about managing and replicating data on-premise. 
+
+## What you'll build
+
+This example depicts you how to use AmazonS3 connector to 
+
+1. Create a S3 bucket (a location for storing your data) in Amazon cloud 
```

Comment:
Add a period at the end of each bullet point or list number.

Reply:
Updated.